browser/images/lc_square.svg

```diff
@@ -1,11 +0,0 @@
-<?xml-stylesheet type="text/css" href="icons.css" ?>
-<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-  <path
-	 class="icn icn--shape-color" 
-     d="m2.5 2.5v19h19v-19z"
-     fill="#fafafa" 
-	 stroke="#3a3a38" 
-	 stroke-linecap="round" 
-	 stroke-linejoin="round"
-     />
-</svg>

```

Comment:
This SVG file doesn't appear to be mentioned in the aliases anywhere. Was it meant to be deleted?